### PR TITLE
Missing menu entries when going to Jobs, Events #167

### PR DESCRIPTION
**Pull request by evalica**
- No links are displayed, so adding a local global.yml
- Removing .logo class from svg

### CODE REVIEW

**Comment by evalica** [comment]
In the future we should try to remove this duplication of header.html and data files

File: _includes/header.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
             </button>
             <a class="navbar-brand page-scroll" href="/#page-top">
                 <span class="logo">
-                    <img src="//opensourcedesign.net/images/opensourcedesign-logo.svg" class="logo" alt="Open Source Design Logo"/>
+                    <img src="//opensourcedesign.net/images/opensourcedesign-logo.svg" alt="Open Source Design Logo"/>
```
Comment (jancborchardt) [inline _includes/header.html]: We don’t need this alt text, as the logo is purely cosmetic. It says "Open Source Design" in the next line, so we can use `alt=""` here. :)

**Comment by elioqoshi** [comment]
Looks good to me, thanks for the PR! I will leave it to someone else to give it a last look and eventually merge it :)

**Comment by evalica** [comment]
So it's finally fixed! YEY :) Waiting for the approval / merge of https://github.com/opensourcedesign/events/pull/123 so I can close https://github.com/opensourcedesign/opensourcedesign.github.io/issues/167

**Pull request merged**